packages/cli/test/fixtures/unit/commands/build/with-cron/api/custom.js

```diff
@@ -0,0 +1,20 @@
+export const config = {
+  runtime: 'edge',
+  cron: {
```

Comment:
I have vague memories of people saying we should move from export const config to independent export const runtime, export const cron. Perhaps it's no more planned to do so?